js/exmo.js

```diff
@@ -375,6 +375,8 @@ module.exports = class exmo extends Exchange {
                 'name': name,
                 'type': type,
                 'active': active,
+                'deposit': undefined,
```

Comment:
Please, check the comment above, each currency has a `withdraw` `enabled: true` flag and a `deposit` `enabled: true` flag, those are handled as `activeProvider`, let's put those values into the returned structure.

Reply:
I'll add deposit / withdraw in each provider.

Reply:
Can you implement the same logic as we do with the other exchanges here?

Reply:
Okay, updated!